app/views/questions/show.html.erb

```diff
@@ -12,6 +12,11 @@
         <br>
         <div id="question-body" >
           <%= @question.body %><br>
+          <div id="delete-question">
+              <% if !current_user.nil? && current_user.id == @question.user_id %>
```

Comment:
I tried this and the page fails to render on a nil error if the user isn't logged in.